launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -104,6 +106,18 @@
     </include>
   </group>
 
+  <!-- Planning Evaluator -->
+  <group if="$(var launch_planning_evaluator)">
+    <include file="$(find-pkg-share planning_evaluator)/launch/planning_evaluator.launch.xml"/>
+  </group>
```

Comment:
@TakaHoribe I remember you saying that planning_evaluator nodes are expected to launch regardless of simulation or real vehicle driving (e.g., to publish diagnostics for MRM triggering). Is it correct?
If that's true, then I think this should be included from planning.launch, not simulator.launch

Reply:
@kyoichi-sugahara @mitsudome-r Thank you for the mention. Right, the planning evaluator should be launched in both sim and real, and thus be called from the planning.launch

Reply:
@mitsudome-r @TakaHoribe 
I have modified code so that the `planning_evaluator` node is always laucnhed from planning.launch and confirmed its behavior with this new [PR](https://github.com/autowarefoundation/autoware_launch/pull/240).